beamer/tests/util.py

```diff
@@ -167,26 +167,27 @@ def make_request(
     target_chain_id=None,
     zero_fees=True,
 ) -> RequestId:
-    if token.balanceOf(requester) < amount:
-        token.mint(requester, amount, {"from": requester})
+    if zero_fees:
```

Comment:
I find it weird that `make_request` would have the side effect of bringing the fees to 0. Maybe restore them to their initial value at the end of the function.

Reply:
Fully agreed. Will do that in a separate PR.